devito/builtins/arithmetic.py

```diff
@@ -58,6 +57,58 @@ def norm(f, order=2):
     return f.dtype(v)
 
 
+@dv.switchconfig(log_level='ERROR')
+def sum(f, dims=None):
```

Comment:
how about we call this `reduce` and add an optional argument `op='+'` ? no need to implement the other cases in this PR, OC

Reply:
ping

Reply:
It's a bit of a pain cuz these reudction (+, -, ...) are not easy to carry around but would be a good extension at some point yes

Reply:
we could have thrown a NotImplementedError for all but +, but okay can postpone to another PR, no big deal